editor/resources/quick-help-view.fxml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.image.Image?>
+<?import javafx.scene.image.ImageView?>
+<?import javafx.scene.layout.*?>
+<VBox id="quick-help-box" alignment="CENTER" spacing="10" visible="false" AnchorPane.bottomAnchor="0.0" AnchorPane.leftAnchor="0.0" AnchorPane.rightAnchor="0.0" AnchorPane.topAnchor="0.0" xmlns="http://javafx.com/javafx/17.0.2-ea" xmlns:fx="http://javafx.com/fxml/1">
+    <ImageView fitHeight="150.0" fitWidth="200.0" pickOnBounds="true" preserveRatio="true">
+        <Image url="@logo-ver-outline-dark-320.png" />
```

Comment:
I think we should use the SVG version of the logo. Perhaps the white one with some transparency could work? You can get it in SVG format from here: https://defold.com/logo-and-trademark/

Reply:
Does javafx support display svg natively?

For now, I change to use the Defold logo `logo-ver-outline-white.png`, and it looks pretty good.

Reply:
I think it does. Did you try it?

Reply:
Yes, I already tried. It display nothing when we use svg format.

![image](https://github.com/defold/defold/assets/6185205/9ecc914c-b984-409b-b768-c2e15c0107c1)

Reply:
Hmm, yes. It seems it supports SVG shape syntax using the `<SVGPath>` element, but not the full SVG spec. Oh well.